cmd/tailscale/cli/web.html

```diff
@@ -99,6 +99,9 @@ <h3 class="text-3xl font-semibold mb-3">Log in</h3>
 		<a href="#" class="mb-4 link font-medium js-loginButton" target="_blank">Reauthenticate</a>
 	</div>
 	{{ end }}
+	<div class="text-right">
```

Comment:
Putting it inside the white card on the right hand side makes it feel…

1. Related to the other Tailscale content inside the card
2. Prominent because of all the negative space around it

<img width="400" alt="image" src="https://user-images.githubusercontent.com/303731/189712189-16a5930e-fb8d-49c6-a2a2-fe910f132b59.png">

Instead, I think this would work better underneath the card as a little footer.

<img width="400" alt="Screenshot 2022-09-12 at 12 55 02@2x" src="https://user-images.githubusercontent.com/303731/189712308-12843979-02de-403d-9651-a4f31cd8d5ba.png">

You can do this by adding this HTML block under the `</main>` line below:

```html
<footer class="container max-w-lg mx-auto text-center">
	<a class="text-xs text-gray-500 hover:text-gray-600" href="{{ .LicensesURL }}">Open Source Licenses</a>
</footer>
```

And then just add a `mb-8` to the `class` attribute on the `main` up on line 14

Reply:
done

<img width="635" alt="Screen Shot 2022-09-12 at 10 15 19 AM" src="https://user-images.githubusercontent.com/1112/189715870-d08f59f2-83fe-42c9-9756-d2092b3dcacf.png">